### PR TITLE
Thanking Potato for its contribution.

### PR DESCRIPTION
**Pull request by rileyjenk**

### CODE REVIEW

**Review by cnharris10 — APPROVED** [review]
All the ✅s

**Comment by ijurica** [comment]
This PR is so to the point. 
I just can't stop laughing

**Review by ijurica — APPROVED** [review]

**Review by ljadvey — APPROVED** [review]

File: specification/potato.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Potato
+
+No potato was harmed during the production of this specification. We thank potato for its contribution in the creation of this specification.
```
Comment (aqu-erp) [inline specification/potato.md:3]:
```suggestion
No starchy root vegetables were harmed during the production of this specification. We thank potato for its contribution in the creation of this specification.
```
Minimizing use of term in its own description. Very serious.

Comment (rileyjenk) [reply specification/potato.md]: Totally doing this. Added out of this commit however because we changed a few things.

Comment (rileyjenk) [reply specification/potato.md]: @aqu-erp check it out now

**Review by flanakin — APPROVED** [review]

**Comment by flanakin** [comment]
I feel like we need Sean's sign-off 🙃

**Review by udam-f2 — APPROVED** [review]

**Pull request merged**